promscale/quick-start.md

```diff
@@ -6,40 +6,31 @@ Jaeger and OpenTelemetry. It also includes some additional pre-configured toolin
 to get familiar with additional Promscale features, such as application performance
 monitoring dashboards and visualization tools.
 
-To learn more about how Promscale can improve your observability stack, see
-[About Promscale][about-promscale]. For instructions on installing Promscale in your
+For instructions on installing Promscale in your
 production or staging environments, see [Installing Promscale][install-promscale].
 
 Before you begin, make sure that you have [installed Docker Compose][docker-compose].
 
-## Install Promscale with Docker Compose
-To install Promscale with Docker Compose, you need to start by cloning the Promscale 
-[repository][gh-promscale], and then use the Docker Compose tool to install it. The 
-`docker-compose` directory contains services for these components:
-* Promscale for analytics and long term storage of metrics and traces
-* Prometheus with the node exporter to generate and collect metrics
-* A microservices application and the OpenTelemetry Collector to generate and collect traces
-* Grafana and Jaeger to visualize metrics and traces
-
 ### Installing Promscale with Docker Compose
 
 <procedure>
 
-1. Clone the repository that defines the services for Promscale:
+1. Clone the repository that defines the services for Promscale, change to
+   the `promscale-demo` directory and use docker compose to run Promscale:
```

Comment:
Steps really need to be atomic. By turning this into a three-step step, you're just making the procedure more complicated because you're increasing the cognitive load on the reader for each step. For example, imagine I was giving you a recipe. If I was to say "Get the eggs and the flour and mix them together with the milk", that's way more complicated than saying "1. Mix the eggs and the flour in a bowl. 2. Add in milk and stir well." There's a very good reason we don't write recipes like that, and it holds true for more technical things as well.

Reply:
I get what you are suggesting. But the user doesn't have to actually understand what each step means as it's all bout cloning, traversing, and setting up the env, this makes the user copy and pastes 3 times. FYI, I also did some user testing with developers to collect feedback.

Reply:
Sure, but in that case, it's not a procedure. If you boil this whole document down to one (albeit complicated) step, what is the point of this document?

Reply:
We don't need to call this a procedure if you don't want to. For installation, a procedure with one step is most of the times better than a procedure with several steps. Users don't care so much about that. I've tested this in the past while at New Relic and conversions did improve.

What is the point of having three steps for this? This is not something we expect users to have to do again and again, it's do it once as quickly as possible and you can then forget. It's also not doing any important changes to their system. Imagine all that happened within a script we created. Instead of doing three steps, you would just run the script. Here is the same but we don't need to create an maintain a script for that.

In my past experience we also see significant increases in conversion when users just copied and pasted everything to accomplish a task in one command. 

Most observability products I know to offer a one command experience to deploy: tobs, Pixie, Datadog, New Relic, Dynatrace, etc. I don't know why this is controversial?

Our goal with this document is
1. Give users a very fast way to deploy a test environment so they can try the product (since we don't have a live environment we host where they can do it, which is something we are considering). They won't know what to do unless we tell them.
2. Guide them while they navigate through the data. And through that navigation we could teach them and show them what Promscale is and how it works.

Does it make sense?